2.0.0/amd64/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get update && \
       netbase \
       unzip \
       wget \
+      fontconfig \
```

Comment:
Hehe, a big note at the top of the file doesn't really help, if you don't read it :P I'll update my commit!